File: git-fundamentals/exercises/jvm/src/test/kotlin/com/willmadison/fun/FunTest.kt
package com.willmadison.`fun`

import org.assertj.core.api.AssertionsForInterfaceTypes.assertThat
import org.junit.Before
import org.junit.Test

class FunTest {

    private lateinit var speaker: Fun

    @Before
    fun setUp() {
        speaker = Fun()
    }

    @Test
    fun `it should speak when given an name`() {
        assertThat(speaker.Say("Ms. Hannigan")).isEqualTo("Hello, Ms. Hannigan")
    }
}

File: git-fundamentals/exercises/javascript/test/fun_test.js
var assert = require('assert');
var fun = require('../fun')

describe('Fun', function () {
    describe('#speak()', function () {
        it('should return Hello, ${name} when given a name', function () {
            assert.equal(fun.speak('Ms. Hannigan'), 'Hello, Ms. Hannigan');
        });
    });
});